test/integration_test.go

```diff
@@ -198,7 +198,7 @@ func TestLogCustomFormatOutput(t *testing.T) {
 		absPathReg = `(?:/[^/]+)*/` + regexp.QuoteMeta(testFixtureLogFormatter)
 	)
 
-	testCases := []struct {
+	tc := []struct {
```

Comment:
test/integration_test.go:201:2: variable name 'tc' is too short for the scope of its usage (varnamelen)
        tc := []struct {